build/makelib/golang.mk

```diff
@@ -91,8 +91,6 @@ endif
 # NOTE: the install suffixes are matched with the build container to speed up the
 # the build. Please keep them in sync.
 
-# we run go build with -i which on most system's would want to install packages
```

Comment:
Why is this gone?

Reply:
`-i` is deprecated in 1.16. The [go 1.16 release notes](https://golang.org/doc/go1.16) description for the deprecation says the build cache already helps since 1.10.
```
The -i flag accepted by go build, go install, and go test is now deprecated. 
The -i flag instructs the go command to install packages imported by packages named on the command line. 
Since the build cache was introduced in Go 1.10, the -i flag no longer has a significant effect on build times, 
and it causes errors when the install directory is not writable.
```